model/project_ref.go

```diff
@@ -2203,6 +2242,53 @@ func (p *ProjectRef) GetGithubProjectConflicts() (GithubProjectConflicts, error)
 	return res, nil
 }
 
+// ValidateProjectCreation returns an error if the total or owner/repo project limit set by the admin settings
+// has been reached. If the project is not enabled, it will grip a warning but not error.
```

Comment:
`If the project is not enabled, it will grip a warning but not error.`
Is this being done in this function, or is that how we expect the function to be used?